### PR TITLE
Add a test and temporarily disable some plugin tests.

### PR DESCRIPTION
**Pull request by monopole**
@Shell32-Natsu

### CODE REVIEW

File: api/go.mod [hunk 1]
```diff
@@ -19,3 +19,5 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.4.1
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace sigs.k8s.io/kustomize/kyaml v0.4.1 => ../kyaml
```
Comment (Shell32-Natsu) [inline api/go.mod:23]: Why do we add a replace for kyaml here? I don't see any change in kyaml for this PR.

Comment (monopole) [reply api/go.mod:23]: it's just for continued development.
I could do it in a different PR if you prefer, just being lazy

**Comment by Shell32-Natsu** [comment]
lgtm

**Comment by monopole** [comment]
Thanks.

Now I'm going to work on removing use of the api/transform package.
Once that's gone, large parts of UnstructAdapter can be dropped, rather than converted to kyaml

**Pull request merged**